feat: ansible 2.15.1

gonzaloetjo opened the pull request:
<!-- Thank you for sending a pull request! Please make sure:
1. Your PR fixes a referenced issue, please create one if no issue applies to your PR.
2. The issue number is referenced in the branch name.
3. You follow the contributing rules at: https://github.com/TOSIT-IO/tdp-collection/blob/master/docs/contributing.md.
-->

#### Which issue(s) this PR fixes

<!-- Example: "Fixes #(issue number)" or "Fixes (link of issue)". -->

Fixes #280 and #263

#### Additional comments

- Use ansible-core 2.15.1
- Install collection dependencies on `ansible_dependencies` through the setup script and `requirements.yml`
<!-- Example: "I was not sure if it is the right way to do but..." -->



#### Agreements

<!-- To make clear that you license your contribution under the Apache License Version 2.0, January 2004 (http://www.apache.org/licenses/LICENSE-2.0) and that you give permission to TOSIT (https://www.tosit.io/), you have to acknowledge this by using the following check-box. -->

- [x] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0).
- [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

kpgtek commented on requirements.txt:
```diff
@@ -1,4 +1,3 @@
-ansible==2.9.27
+ansible-core==2.15.1
 jmespath==0.10.0
```
```jmespath``` is a requirement for Prometheus.
If we use Python 3.9, I don't know if this version 0.10.0 would be compatible.
On my system with Python 3.9, the version 1.0.1 is installed.

gonzaloetjo replied:
I added the 1.0.1 version

PACordonnier commented:
I dont think we should accept the PR as is. The instructions in the README are not in sync with the current configured submodules (even after https://github.com/TOSIT-IO/tdp-getting-started/pull/286). For example `tdp plan dag` is not implemented in stable.

I understand the need of the PR (add instructions for the current stable tdp-lib and ansible 2.15) but it should include the bump in the submodules as well to work.

I suggest we this PR to a dev branch, this way people who `git clone` master will have a working version.

gonzaloetjo commented:
@rpignolet @PaulFarault I added the `dev` branch where this should be merged. Also added a note on the top of `README.md` warning that it's a dev branch and some components may not work correctly (currently it's server and ui, but may be something different at an other point).
